spec/models/digital_object/base_spec.rb

```diff
@@ -97,6 +98,7 @@
             pending_unpublish_from: [],
             structured_children: { 'type' => 'sequence', 'structure' => [] },
             dynamic_field_data: {},
+            rights: {},
```

Comment:
Should we add a test to make sure a populated json get stored? or do we want to wait to implement rights validation before we do that?

Reply:
I want to rethink digital object validation overall, so I'm good with waiting until we add rights validation.  Right now it's sort of covered by the general tests for any generically defined `metadata_attribute`.

Reply:
👍